js/interface.js

```diff
@@ -2472,10 +2476,10 @@ function checkSubmissionStatus(origin, iosSubmissions) {
 
       build.id = submission.id;
       build.updatedAt = ((submission.status === 'completed' || submission.status === 'failed' || submission.status === 'cancelled' || submission.status === 'ready-for-testing' || submission.status === 'tested') && submission.updatedAt) ?
-        moment(submission.updatedAt).format('MMM Do YYYY, h:mm:ss a') :
+        moment(submission.updatedAt).format('MMM Do YYYY, HH:mm') :
```

Comment:
@squallstar Is this used for display? Just checking because we want most dates displayed in 12h format with am/pm.

Reply:
@tonytlwu sorry I've reverted it, I had this change locally from some time ago (not sure, maybe someone asked me to change this)